docs/pages/Theming/ColorTokens/Basics.md

```diff
@@ -0,0 +1,47 @@
+# Using Color Tokens
+
+With the Fluent Design System, we have a set of colors that we have agreed to use for our UI.
+This page describes how to access these colors.
+
+## Global
+
+Our set of global tokens is the complete set of the colors that we agree to use for our UI. Each color has a specific name that is assigned to a hex value. For example, `grey74` is `#bdbdbd`.
+These names and values do not change no matter the platform or theme.\*
```

Comment:
Should this `\*` be here?

Reply:
Yeah, the `\` is an escape character for the `*` to show up as an asterisk, and not as a bullet (which is what asterisks usually produce in markdown)

The intent is to show that there's an exception or caveat to that statement.